.test-env

```diff
@@ -0,0 +1,40 @@
+# Configs for python sdk docker:
+PYTHON_VERSION=${PYTHON_VERSION:-3.8}
+
+# Configs for testing repo download:
+SDK_TESTING_URL="https://github.com/algorand/algorand-sdk-testing"
+SDK_TESTING_BRANCH="trim-indexer"
+SDK_TESTING_HARNESS="test-harness"
+
+# Used to determine sandbox build type:
+TYPE="channel" # OR "source"
+
+# Used when TYPE==channel
+ALGOD_CHANNEL="nightly"
+
+# Used when TYPE==source
+ALGOD_URL="https://github.com/algorand/go-algorand"
+ALGOD_BRANCH="feature/avm-box"
+ALGOD_SHA=""
+
+# Used regardless of the TYPE:
+NETWORK_TEMPLATE="images/algod/DevModeNetwork.json"  # refers to the ./images directory in the sandbox repo
+NETWORK_NUM_ROUNDS=30000
+INDEXER_URL="https://github.com/algorand/indexer"
+NODE_ARCHIVAL="False"
+INDEXER_BRANCH="tzaffi/box-search"
+INDEXER_SHA=""
+
+# Sandbox configuration:
+SANDBOX_URL="https://github.com/algorand/sandbox"
+SANDBOX_BRANCH="master"
+LOCAL_SANDBOX_DIR=".sandbox"
+
+# replacement values for Sandbox's docker-compose:
+ALGOD_CONTAINER=sdk-harness-algod
+KMD_PORT=60001
+ALGOD_PORT=60000
+INDEXER_CONTAINER=sdk-harness-indexer
+INDEXER_PORT=59999
+POSTGRES_CONTAINER=sdk-harness-postgres
+POSTGRES_PORT=65432
```

Comment:
@tzaffi Leaving a _suggestion_ as a visual reminder following up on our verbal.  Prior to merging, my request is to remove algorand-sdk-testing overrides.

For the benefit of other reviewers:  While open to discussion, my view continues to be:
* `.env` in algorand-sdk-testing governs configuration across SDKs.  Consequently, I expect it'll be rare to merge `.test-env` to `develop` with algorand-sdk-testing overrides.
* I see local testing and/or one-off SDK PRs where `.test-env` overrides are a welcomed convenience. 
* On balance, I think the PR adds a useful + flexible facility that requires intentional usage to minimize misconfigurations.

```suggestion
```

Reply:
Will probably take the suggestion.

Reply:
I've introduced a new variable `OVERWRITE_TESTING_ENVIRONMENT=0` which allows for overwriting, along with a warning to the user.